opaque_keys/edx/locator.py

```diff
@@ -1330,3 +1330,67 @@ def to_deprecated_list_repr(self):
 
 # Register AssetLocator as the deprecated fallback for AssetKey
 AssetKey.set_deprecated_fallback(AssetLocator)
+
+
+class AggregateCourseLocator(AggregateCourseKey):    # pylint: disable=abstract-method
```

Comment:
@cpennington @clintonb  The only format need is `org+course`. I have added/update tests in `opaque_keys/edx/tests/test_locators.py` for the `AggregateCourseLocator`.

I was unable to directly get a `AggregateCourseKey` locator from a `CourseKey` locator mainly due to changed `KEY_TYPE` and [`OpaqueKey` Hierarchy](https://github.com/edx/edx-platform/wiki/Opaque-Keys-(Locators)#opaquekey-hierarchy). To handle the conversion from `CourseKey` to `AggregateCourseKey` I have added a new method `from_course_key` in `AggregateCourseKey` which takes a course key object and returns a locator `AggregateCourseLocator` from it. Please let me know your suggestion on this approach.